### PR TITLE
[Sprint3] Feature create ui shop detail

### PR DESCRIPTION
**Pull request by nguyenhuy95dn**
Feature create ui shop detail 
Link redmine: https://dev.framgia.com/redmine/issues/114085
Task: Detail Shop
Effect: N/A

### CODE REVIEW

File: app/src/main/java/com/framgia/forder/screen/DetailShop/DetailShopFragment.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.framgia.forder.R;
+import com.framgia.forder.databinding.FragmentDetailShopBinding;
+
+/**
+ * DetailShop Screen.
+ */
+public class DetailShopFragment extends Fragment {
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/DetailShop/DetailShopFragment.java]: change to ShopDetail

File: app/src/main/java/com/framgia/forder/screen/listshop/ListShopViewModel.java [hunk 2]
```diff
@@ -52,7 +55,8 @@ public void onGetListAllShopSuccess(List<Shop> shops) {
 
     @Override
     public void onItemRecyclerViewClick(Object item) {
-        //TODO: Click Item
+        mNavigator.goNextChildFragment(R.id.layout_content, ShopDetailFragment.newInstance(), true,
+                Navigator.RIGHT_LEFT, TAG);
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/listshop/ListShopViewModel.java]: Change to
```java
mNavigator.goNextChildFragment(R.id.layout_content, ShopDetailFragment.newInstance(), true,
                 Navigator.RIGHT_LEFT, "ShopDetailFragment");
```

File: app/src/main/java/com/framgia/forder/screen/listshop/ListShopViewModel.java [hunk 3]
```diff
@@ -14,6 +16,7 @@
 
 public class ListShopViewModel extends Observable implements ListShopContract.ViewModel,
         BaseRecyclerViewAdapter.OnRecyclerViewItemClickListener<Object> {
+    private static final String TAG = "ShopDetailFragment";
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/listshop/ListShopViewModel.java]: remove

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -32,4 +32,16 @@
         <item name="android:windowBackground">@drawable/background_splash</item>
     </style>
 
+    <style name="StyleInfoShop">
```
Comment (daolq3012) [inline app/src/main/res/values/styles.xml]: change name to ``StyleContainerShopDetail``

File: app/src/main/res/values/styles.xml [hunk 5]
```diff
@@ -32,4 +32,16 @@
...
+        <item name="android:layout_marginLeft">@dimen/dp_5</item>
+        <item name="android:layout_marginRight">@dimen/dp_5</item>
+        <item name="android:layout_marginTop">@dimen/dp_5</item>
+    </style>
+
+    <style name="StyleImageShopInfo">
```
Comment (daolq3012) [inline app/src/main/res/values/styles.xml]: change name to ``StyleImageShopDetail``

File: app/src/main/res/layout/fragment_shop_detail.xml [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+                    >
+                    <ImageView
+                        android:layout_width="wrap_content"
+                        android:adjustViewBounds="true"
+                        android:scaleType="centerCrop"
+                        android:src="@drawable/ic_time_shop"
+                        style="@style/StyleImageShopInfo"
+                        />
+                    <TextView
+                        android:layout_width="wrap_content"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_shop_detail.xml]: pls create a style for nomarl text view
 
android:layout_width="wrap_content"
android:layout_height="wrap_content"
android:textColor="@color/color_black"
android:textSize="@dimen/sp_17"

File: app/src/main/res/layout/fragment_shop_detail.xml [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+                        />
+                </LinearLayout>
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="@dimen/dp_50"
+                    android:background="@android:color/white"
+                    style="@style/StyleInfoShop"
+                    >
+                    <ImageView
+                        android:id="@+id/imageView2"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_shop_detail.xml]: pls check convention

File: app/src/main/res/layout/fragment_shop_detail.xml [hunk 8]
```diff
@@ -0,0 +1,245 @@
...
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    >
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.framgia.forder.screen.shopdetail.ShopDetailViewModel"
```
Comment (daolq3012) [inline app/src/main/res/layout/fragment_shop_detail.xml]: change package name from ``shopdetail``  to ``shopDetail``

**Pull request merged**